### PR TITLE
test(model): add serialization test to model

### PR DESCRIPTION
**Pull request by lvamaral**
What: add basic unit test to model
Why: ensure our model serializes movies with appropriate fields

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,4 +9,5 @@ before_script:
   - yarn run db:reset
 script:
   - yarn test
+  - yarn run enforce
```
Comment (pallavi) [inline .travis.yml]: nit: you don't need `run` after yarn commands. you can change these to `yarn enforce` and `yarn lint`

**Review by pallavi — APPROVED** [review]
LGTM!
add `coverage` to your `.gitignore` file to get rid of all these autogenerated coverage files

File: .travis.yml (hunk 1 above)
Comment (lvamaral) [reply .travis.yml]: done! ty

**Review by jesscxu — APPROVED** [review]
LGTM

**Pull request merged**